pages/windows/fc.md

```diff
@@ -0,0 +1,32 @@
+# fc
+
+> Compare the differences between two files or sets of files.
```

Comment:
Compare the differences? Does this mean that it compares multiple diffs?

Reply:
This is the output from the command.
![](https://image.prntscr.com/image/mfNWfZNJQjKK3skLB3qxuQ.png)